### ISSUE TITLE
Method calls on literal strings are not moved to the next line; may exceed line width

### ISSUE THREAD

**Issue by bethcutler** [facebook/ktfmt#256]
Calling `trimMargin()` on a long literal can exceed the line width.
```
"""This string has approximately eighty-four characters which means trimMargin overruns""".trimMargin()
```

Instead, we would expect the call to wrap on the next line:
```
"""This string has approximately eighty-four characters which means trimMargin overruns"""
    .trimMargin()
```

**Comment by bethcutler** [comment]
FYI, this is likely related to #237

**Comment by hick209** [comment]
This looks fixed as of v0.47, since it formats:
```
"""This string has approximately eighty-four characters which means trimMargin overruns""".trimMargin()
```
to
```
"""This string has approximately eighty-four characters which means trimMargin overruns"""
    .trimMargin()
```
